app/controllers/api_controller.rb

```diff
@@ -21,15 +21,15 @@ def authenticate_with_jwt!
     return @current_user = User.first if Rails.env.development?
 
     payload = JwtService.decode(token: token)
-    if DateTime.parse(payload['expiration_date']) <= DateTime.now
-      render json: {error: 'Auth token has expired. Please login again'}, status: 401
+    if DateTime.parse(payload['expiration_date']) <= DateTime.current
+      render json: {error: 'Auth token has expired. Please login again'}, status: :unauthorized
     elsif store_owner_code
       @current_user = User.where(store_owner_code: store_owner_code, role: :store_owner).first
```

Comment:
didn't rubocop complain of not using find_by?

Reply:
(I got it on another project)

Reply:
Not sure why it didn't complain in this instance. It did complain somewhere else :thinking:

Reply:
The line, in `app/models/store.rb`, errors out:

```
StatusCrowdsourceUser.where(store_id: id).first
```

but if I add the same line to this controller, it doesn't. Not sure what's happening.

Reply:
weird :-o

Reply:
This specific cop only runs in model files:
https://github.com/rubocop-hq/rubocop-rails/blob/master/config/default.yml#L221

Reply:
@simaob : I have pushed another commit with two more where.first -> find_by transforms